tests/simulator/test_advance.py

```diff
@@ -237,6 +240,120 @@ def test_field_coarsening_via_subcycles(self, refinement_boxes):
         self._test_field_coarsening_via_subcycles(dim, interp_order=1, refinement_boxes=refinement_boxes)
 
 
+
+
+    def _test_field_level_ghosts_via_subcycles_and_coarser_interpolation(self, ndim, interp_order, refinement_boxes, refinement_ratio=2):
+        """
+          This test runs two virtually identical simulations for one step.
+            L0_datahier has no refined levels
+            L0L1_datahier has one refined level
+
+          This is done to compare what would be the uncoarsened L0 values in L0L1_datahier
```

Comment:
"uncoarsened L0 values" is misleading, since it's L1 values that are coarsened onto L0, and L0 domain nodes overlapped by L1 that are updated via coarsening....

Reply:
"L0 values that haven't received the coarsened values of L1 because there is no L1"